package.json

```diff
@@ -100,7 +100,7 @@
     "detect-port": "^1.2.2",
     "dirty-chai": "^2.0.1",
     "go-ipfs-dep": "0.4.13",
-    "ipfs": "^0.27.5",
+    "ipfs": "github:ipfs/js-ipfs",
```

Comment:
Let's old this PR until we have a new release of js-ipfs (it can happen soon).

Reply:
It's here, can't wait to get this in :)

<img width="792" alt="screen shot 2018-02-21 at 12 38 57 pm" src="https://user-images.githubusercontent.com/1094341/36498558-3da5f07c-1704-11e8-83a1-4a8066e7ae10.png">